source/GameInterface/Services/Clans/Patches/ClanChangeInfluencePatch.cs

```diff
@@ -17,8 +17,9 @@ public class ClanChangeInfluencePatch
 
         static bool Prefix(Clan clan, float amount)
         {
-            if (PolicyProvider.AllowOriginalCalls) return true;
+            if (amount == 0f) return false;
```

Comment:
move `if (AllowedInstance.IsAllowed(clan)) return true;` after this line. if we want to allow this call then it should be one of the first checks